cdc_stream/base_command.cc

```diff
@@ -56,6 +56,30 @@ std::function<void(const std::string&)> BaseCommand::JedecParser(
   };
 }
 
+std::function<void(const std::string&)> BaseCommand::PortRangeParser(
```

Comment:
ParsePortRange and this functions are almost the same. Could you please re-allocate the same code in a separate function and use it in both cases, please?

Reply:
Done. Also added tests.